src/libs/actions/Policy.ts

```diff
@@ -1820,6 +1821,37 @@ function hideWorkspaceAlertMessage(policyID: string) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {alertMessage: ''});
 }
 
+function updateAddress(policyID: string, newAddress: CompanyAddress) {
+    // TODO: Change API endpoint parameters format to make it possible to follow naming-convention
+    const parameters: UpdatePolicyAddressParams = {
+        policyID,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        'data[addressStreet]': newAddress.addressStreet,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        'data[city]': newAddress.city,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        'data[country]': newAddress.country,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        'data[state]': newAddress.state,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        'data[zipCode]': newAddress.zip,
+    };
```

Comment:
100% agree. I think this is something what needs backend changes. I tried many different ways to make this API request and this is the only one which works.

Reply:
The thread about changing the endpoint on backend: https://swmansion.slack.com/archives/C05S5EV2JTX/p1713825690019579
In case of changes we need to modify this part.